src/main/java/Data/UserManager.java

```diff
@@ -0,0 +1,61 @@
+package Data;
+
+import Library.Book;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class UserManager {
```

Comment:
Maybe UserHandler? To be consistent with naming?

Reply:
Done